client/src/components/Play.jsx

```diff
@@ -39,8 +38,10 @@ export default function Play(props) {
       }}>
         {/* <div style={{ fontSize: '24px', color: 'black' }}>Play View</div> */}
         <PlayViewWaveform trackUrl={props.songData.path_to_song} />
-        <div>{props.songData.title}</div>
-        <div>{props.songData.artist}</div>
+        <div>{"Song Title: " + props.songData.title}</div>
+        <div>{"Artist Name: " + props.songData.artist}</div>
+        <div>{"Tags: " + props.songData.tags}</div>
+        {/* <div>{"Tags: " + props.songData.tag_names_string}</div> */}
```

Comment:
We should be putting classNames for styling and querying: `<div className="play-songTitle">`.

Reply:
Yes that's right! These are placeholders for now just to visualize the data on the Play View. Calvin and I will be styling with MUI and removing these this week.